### PR TITLE
(doc) adding tools documentation for job creation

### PR DESCRIPTION
**Pull request by rigelk**

### CODE REVIEW

File: support/doc/tools.md [hunk 1]
```diff
@@ -45,22 +47,24 @@ $ npm run build:server
...
+$ node dist/server/tools/import-videos.js \
+    -u "PEERTUBE_URL" \
+    -U "PEERTUBE_USER" \
+    --password "PEERTUBE_PASSWORD" \
+    -t "TARGET_URL"
 ```
 
- * PEERTUBE_URL : the full URL of your PeerTube server where you want to import, eg: https://peertube.cpy.re
- * PEERTUBE_USER : your PeerTube account where videos will be uploaded
- * PEERTUBE_PASSWORD : password of your PeerTube account (if ommited, you will be prompted for)
```
Comment (Chocobozzz) [inline support/doc/tools.md]: Why did you remove `(if ommited, you will be prompted for)`?

File: support/doc/tools.md [hunk 2]
```diff
@@ -45,22 +47,24 @@ $ npm run build:server
...
- * TARGET_URL : the target url you want to import. Examples:
-   * YouTube:
-     * Channel: https://www.youtube.com/channel/ChannelId
-     * User https://www.youtube.com/c/UserName or https://www.youtube.com/user/UserName
-     * Video https://www.youtube.com/watch?v=blabla
-   * Vimeo: https://vimeo.com/xxxxxx
-   * Dailymotion: https://www.dailymotion.com/xxxxx
+* `PEERTUBE_URL` : the full URL of your PeerTube server where you want to import, eg: https://peertube.cpy.re/
```
Comment (Chocobozzz) [inline support/doc/tools.md]: https://github.com/Chocobozzz/PeerTube/pull/612

File: support/doc/tools.md (hunk 1 above)
Comment (rigelk) [reply support/doc/tools.md]: A mistake :/

**Comment by Chocobozzz** [comment]
:kissing_heart:
